src/test/java/com/aws/greengrass/logmanager/LogManagerServiceTest.java

```diff
@@ -83,6 +83,8 @@
 import static com.aws.greengrass.logmanager.LogManagerService.SYSTEM_LOGS_COMPONENT_NAME;
 import static com.aws.greengrass.logmanager.LogManagerService.SYSTEM_LOGS_CONFIG_TOPIC_NAME;
 import static com.aws.greengrass.logmanager.LogManagerService.UPLOAD_TO_CW_CONFIG_TOPIC_NAME;
+import static com.aws.greengrass.logmanager.model.LogFile.bytesNeeded;
+import static com.aws.greengrass.logmanager.model.LogFileTest.getRandomTestStrings;
```

Comment:
This seems weird importing from another test. Maybe move that method to a LogFileTestUtils class inside the `test` folder

Reply:
Probably keep it for now, if we need more such utils method, then we can create such class?